src/unpackertool/plugins/upx/unfilter.h

```diff
@@ -7,7 +7,9 @@
 #ifndef UNPACKERTOOL_PLUGINS_UPX_UNFILTER_H
 #define UNPACKERTOOL_PLUGINS_UPX_UNFILTER_H
 
-#include "retdec/unpacker/dynamic_buffer.h"
+#include "retdec/utils/dynamic_buffer.h"
+
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.